src/peer_review/views/peer_review_view.py

```diff
@@ -1,10 +1,352 @@
 from rest_framework.viewsets import ModelViewSet
+from rest_framework.decorators import action
 from rest_framework.permissions import (
     IsAuthenticated
 )
+from peer_review.permissions import (
+    IsAllowedToCreateDecision,
+)
+from utils.http import DELETE, POST, PATCH, PUT, GET
+from peer_review.models import (
+    PeerReviewDecision,
+    PeerReview,
+)
+from peer_review.serializers import (
+    PeerReviewDecisionSerializer,
+    PeerReviewSerializer,
+)
+from rest_framework.response import Response
+from reputation.models import Contribution
+from django.contrib.contenttypes.models import ContentType
+from discussion.models import Thread
+from reputation.serializers import DynamicContributionSerializer
 
 
 class PeerReviewViewSet(ModelViewSet):
     permission_classes = [
         IsAuthenticated,
```

Comment:
Might want to update the default permission class so user's wont be able to modify other peer reviews.

A permission class where the user can only create and modify/retrieve their own submissions.

Reply:
Yeah agree.